SCF-Session-6/intSet/IntSet.java

```diff
@@ -0,0 +1,185 @@
+package intSet;
+
+import java.util.Arrays;
+
+/**
+ * immutable class IntSet which is uses an array to represent a set of integers in the range 1-1000
+ * @author Ishita_Mathur
+ *
+ */
+public final class IntSet 
+{
```

Comment:
Exception handling is not used in class.

Reply:
done